uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-request-popup-details-tab.component.js

```diff
@@ -487,9 +477,8 @@ define([
      * @return {String} amount of hours, eg. '7.5'
      */
     function getTimeDifferenceInHours (timeFrom, timeTo) {
-      return (getMomentDateWithGivenTime(timeTo)
-        .diff(getMomentDateWithGivenTime(timeFrom), 'minutes') / 60)
-        .toString();
+      return moment.duration(timeTo)
+        .subtract(moment.duration(timeFrom)).asHours().toString();
```

Comment:
why do return a string? I think a number would be more convenient.

Reply:
Was this not going to be turned into a service?

Reply:
I think we agreed that if this is used in 2 places, this not worth turning it into a service.